js/piwik.js

```diff
@@ -4701,6 +4729,18 @@ if (typeof Piwik !== 'object') {
                     configRequestMethod = method || defaultRequestMethod;
                 },
 
+                activeBulkTracking: function(active) {
+                    enableBulkTracking = !!active;
+
+                    if(enableBulkTracking) {
+                      setInterval(sendBulkTrackingRequests, bulkTrackingDelay);
```

Comment:
I'm wondering if there's a way to implement this without actually checking every 300ms. This can be quite intense for CPU and it can slow down heavy applications when scrolling etc maybe (not really an issue probably). An alternative could be in the `sendRequest` method something like this:

```
if(enableBulkTracking && !force) { 
    bulk.push(request);
   setTimeout(sendBulkTrackingRequests, bulkTrackingDelay);
}
```

Do you know what I mean? This way we would only check on demand. People often leave a browser tab open for quite a while and it would check every 300ms without having to otherwise. I'm not 100% sure if the example that I mentioned would work but something in this direction could be nice.